SU-HSDO/suhumsci
# Sparkbox Sprint 6

# READY FOR REVIEW

## Summary
- Reset Button styles [STN-153](https://sparkbox.atlassian.net/browse/STN-153) | PR #471
- Implement QA feedback [STN-150](https://sparkbox.atlassian.net/browse/STN-150) | PR #472
- People UI Pattern [STN-110](https://sparkbox.atlassian.net/browse/STN-110) | PR #476 
- 3 column layout template and styling (along with some changes to 3 col with image) [STN-155](https://sparkbox.atlassian.net/browse/STN-155) | PR #477 
- HTML table styling and JS [STN-162](https://sparkbox.atlassian.net/browse/STN-162) | PR #478 
- Dropdown Filter Text Styling Bug [STN-167](https://sparkbox.atlassian.net/browse/STN-167) | PR #481 
- People UI Pattern Styling [STN-146](https://sparkbox.atlassian.net/browse/STN-146) | PR #480
- Code climate stylelint fixes | PR #475  

## Need Review By (Date)
asap (2/11)

## Urgency
high

## Steps to Test
- ✅ We've verified that this branch passes the code climate and circle CI tests against the develop branch.

1. See other PRs if you'd like to test locally for those steps.
2. Verify code looking at the file changes here.
3. Go into `humsci_basic` folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
